testsuite/cluster-test/src/main.rs

```diff
@@ -758,9 +758,41 @@ impl ClusterTestRunner {
             .map_err(|e| format_err!("Failed to query stats: {}", e))?;
         self.deactivate_all(&mut stop_effects);
         self.wait_until_all_healthy()?;
+        // Simulate 3 regions us_west(40%), us_east(40%), eu(20%)
```

Comment:
Do you mind moving this part to separate function?
@bors-libra delegate+

Reply:
Done